modules/caddy-automatic-https/templates/Caddyfile.tpl

```diff
@@ -0,0 +1,3 @@
+${hostname}
+
+reverse_proxy localhost:4141
```

Comment:
Maybe we should also template the port - I can imagine a use case where some other side-container is using `4141`